File: Prak604/PRAK604-2210817210021-TRISNACAHYAPERMADI.py
X = 0
Y = 0
huruf1 = list(input (""))
huruf2 = list(input (""))
pjg1 = len(huruf1)
pjg2 = len(huruf2)
if pjg1 != pjg2:
    print ("Panjang kalimat berbeda, pesan palsu")
else :
    for i in range (pjg1):
        if " " in huruf1[i] and " " in huruf2[i]:
            continue
        elif huruf1 [i] == huruf2[i]:
            print("*", end="")
            X += 1
        else:
            print("#", end="")
            Y += 1
    print("\n* = ", X)
    print("# = ", Y)
    if X >= Y:
        print("Pesan Asli")
    else: 
        print("Pesan Palsu")     

File: Prak604/PRAK604-2210817210021-TRISNACAHYAPERMADI.c
#include <stdio.h>
#include <string.h>
int main()
{
    char Kode[100],Pesan[100];
    int i,j;
    scanf("%[^\n]%*c",&Kode);
    scanf("%[^\n]%*c",&Pesan);
    int Pkode=strlen(Kode),Ppesan=strlen(Pesan);
    if(Pkode!=Ppesan){
        printf("Panjang kalimat berbeda, pesan palsu");
    }
    else{
        int i=0,j=0;
        for (int k=0;k<Pkode;k++){
            if (Pesan[k]==' ' && Kode[k]==' '){
                continue;
            }
            else if(Kode[k]!=Pesan[k]){
                printf("#");
                i++;
            }
            else if(Kode[k]==Pesan[k]){
                printf("*");
                j++;
            }
        }
        printf("\n*= %i",j);
        printf("\n#= %i",i);   
        if(i<=j){
            printf("\nPesan Asli");
        }
        else{
            printf("\nPesan Palsu");
        }
    }
    return 0;
}